addons/onboarding/models/onboarding_onboarding.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class Onboarding(models.Model):
+    _name = 'onboarding.onboarding'
+    _description = 'Onboarding'
+
+    name = fields.Char('Name of the onboarding', translate=True)
+    alias = fields.Char('One word alias', translate=False)
+    step_ids = fields.One2many('onboarding.onboarding.step', 'onboarding_id', 'Onboarding steps')
+
+    panel_background_color = fields.Selection([('orange', 'Orange'), ('blue', 'Blue')], default='orange')
+    panel_background_image = fields.Binary("Panel's background image")
+    panel_close_action_name = fields.Char('Action to execute when closing the panel')
+
+    is_per_company = fields.Boolean('Should be done per company?', default=True)
+
+    current_progress_id = fields.One2many(
+        'onboarding.progress', 'onboarding_id', 'Onboarding Progress Record', compute='_compute_progress_id')
+
+    progress_ids = fields.One2many('onboarding.progress', 'onboarding_id', 'Onboarding Progress Records')
+
+    _sql_constraints = [
+        ('name_uniq', 'UNIQUE (name)', 'Onboarding name must be unique.'),
```

Comment:
Any specific reason to make the name unique?
I get why the alias has to be, but not sure about the name.